remix-gql-tailwind/app/components/RepoNavigation/RepoNavigation.tsx

```diff
@@ -26,7 +27,7 @@ function RepoNavigation() {
             return (
               <>
```

Comment:
Because this fragment is the top-level return from the loop, you'll want to put the `key` here. [Keys are the only props that a React Fragment will accept.](https://reactjs.org/docs/fragments.html#keyed-fragments)

Switching to `<React.Fragment key={whateverTheKeyIs}>...</React.Fragment>` should do it.

Also, I don't think you should use `cuid` here. If I'm understanding the library correctly, each call to `cuid()` generates a new, unique id. Which means on each render, every item will receive a new key. This prevents React from knowing which items experienced no change in order or content, necessitating a full re-render of the entire list and decreasing performance. Using the index `i` has a similar problem, since array contents might change order. I recommend using the `crumb` data to create the key. Assuming `crumb` is a string and that each one is unique, ``key={`crumb-${crumb}`}`` would be a good option.